tests/backbeat/tests/api/metrics.js

```diff
@@ -99,66 +144,48 @@ describe.skip('Backbeat replication metrics', function dF() {
                         assert(byteResult - prevDataBytes === 100);
 
                         const throughputSize = body.throughput.results.size;
-                        assert(throughputSize > 0);
+                        assert(throughputSize >= 0);
                         next();
                     });
                 }),
         ], done);
     });
 
-    it('should get metrics when specifying metric type', done => series([
-        next => scalityUtils.putObject(srcBucket, key, Buffer.alloc(100), next),
-        next => scalityUtils.compareObjectsAWS(srcBucket, destBucket, key,
-            undefined, next),
-        next => makeGETRequest('/_/backbeat/api/metrics/crr/all/backlog',
-            (err, res) => {
-                assert.ifError(err);
-                assert.equal(res.statusCode, 200);
-                getResponseBody(res, (err, body) => {
-                    assert.ifError(err);
-                    const keys = Object.keys(body);
-                    assert.strictEqual(keys.length, 1);
-                    assert.strictEqual(keys[0], 'backlog');
+    it('should get object-level progress metrics', done => {
```

Comment:
Looks like this test case is already handled in https://github.com/scality/Zenko/blob/development/0.9/tests/backbeat/tests/api/objectMonitor.js#L39.